### ISSUE TITLE
Would love to see support for Horizon in the upcoming update

### ISSUE THREAD

**Issue by theFisher86** [Dheuster/AFT-FO4Edition#34]
AFT is awesome, Horizon is awesome.  Unfortunately I can't figure out a good way to properly patch AFT since so much is happening via scripts.  Horizon is about to have it's 1.6 update and you're working on your  1.2 update... Would love to see a patch that makes these two mods work together, even if it means just disabling some of the incompatible stuff in AFT.

**Comment by Dheuster** [comment]
I am unaware of any direct incompatibilities. Maybe some game balance issues, but I don't know of any CTD or "cant finish the main game" type conflicts.

**Comment by theFisher86** [comment]
The people in the Horizon Discord seem to think the way the inventory
management is done in AFT ends up deleting the "special" Horizon Companion
weapons.  I'm not super familiar with that aspect of Horizon so I can't
speak on it.  However, the problem I had was when I went to recruit Piper
as a companion, I currently had 3 companions (with a max of 5 companions
set in the AFT Holotape).  Each time I tried to recruit her, or any other
companion after that it just acted like they were recruited, then played
the dialog as if Dogmeat was being dismissed since that NPC was being
recruited.  But then instead of dog meat being dismissed it asked if I
wanted to send Piper (or whatever NPC/Companion I tried to recruit) to a
Settlement.  So it got into a loop where trying to have any companion other
than the ones I currently had would just try to recruit them -> Dialog like
a current companion was being dismissed -> Asking which settlement the
person I as trying to recruit should be sent to -> that person is never
recruited.

When I discussed this in the Horizon Discord the general consensus was that
there's some stuff in AFT that "messes" with Horizon and causes all sorts
of weird bugs.  I unfortunately don't know enough about Horizon and can't
figure out enough about AFT from FO4Edit to figure out what may be
conflicting.  It seems AFT does A LOT of stuff via scripts so FO4Edit
patching probably won't do a lot of good there.  Would be awesome if you
could work with some of the Horizon devs to make a full compatibility thing
b/c it'd make for an amazing playthrough,
